bycycle/burst/utils.py

```diff
@@ -108,6 +109,28 @@ def recompute_edges(df_features, threshold_kwargs):
                             enumerate(burst_edges)])
 
     # Recompute is_burst
+    for idx, cyc_idx in enumerate(burst_edges):
+
+        if idx % 2 == 0:
+            direction = 'next'
+        else:
+            direction = 'last'
+
+        # Slice edges rows and recompute burst features
+        lower = cyc_idx-1 if cyc_idx-1 >= 0 else 0
+        upper = cyc_idx+2 if cyc_idx+2 < len(df_features_edges) else len(df_features_edges)
```

Comment:
similar idea as last comment. something like `min(cyc_idx+2, len(df_features_edges))`

Reply:
That's a much simpler than what I have, thanks!